src/google-maps/map-transit-layer/map-transit-layer.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 // Workaround for: https://github.com/bazelbuild/rules_nodejs/issues/1265
-/// <reference types="google.maps" />
+/// <reference types="google.maps" preserve="true" />
```

Comment:
Was this caught by some integration tests?

Reply:
Yeah although the failure was a bit weird and took a while to track down. It was caught by the API extractor not being able to follow the type of the global variables.